config/rbac/auth_proxy_service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
```

Comment:
Optional nitpick: the service could be placed somewhere else than in rbac. Maybe in the default folder?

Reply:
Not sure if this should be done. All the files in the config dir is generated by the operator-sdk and I believe they expect in the subcommands that dir structure would be same to the one they create with different content (if required)

Reply:
Aha, ok. I didn't know that this was generated.